docs/src/main/asciidoc/scheduler-reference.adoc

```diff
@@ -260,6 +260,26 @@ void everyTwoSeconds() { }
 ----
 <1> The config property `myMethod.delay.expr` is used to set the delay.
 
+[[delayed_execution]]
+=== Delayed Execution
+
+`@Scheduled#executionMaxDelay()` can be set to delay each execution of a scheduled method.
+The value represents the maximum delay between the activation of the trigger and the execution of the scheduled method.
+The actual delay varies randomly over time but it never exceeds the maximum value.
+
+The value is parsed with `DurationConverter#parseDuration(String)`.
+It can be a property expression.
+In that case, the scheduler attempts to use the configured value instead: `@Scheduled(executionMaxDelay = "${myJob.maxDelay}")`.
+Additionally, the property expression can specify a default value:
+`@Scheduled(executionMaxDelay = "${myJob.maxDelay}:500ms}")`.
+
+[source,java]
+----
+@Scheduled(every = "2s", executionMaxDelay = "500ms") <1>
+void everyTwoSeconds() { }
+----
+<1> The delay will be a value between 0 and 500 milliseconds. As a result, the period between to `everyTwoSeconds()` executions will be roughly between two and two and a half seconds.
```

Comment:
```suggestion
<1> The delay will be a value between 0 and 500 milliseconds. As a result, the period between to `everyTwoSeconds()` executions will be roughly between one an dhalf and two and a half seconds.
```

You can have first execution delayed by 500ms and the subsequent by 0ms, so it is in fact 1.5s to 2.5s. Or am I missing something? :)